### DISCUSSION TITLE
Getting error: "ELIFECYCLE  Command failed with exit code 1." while trying to deploy.

### DISCUSSION THREAD

**Discussion by tobyab** [vercel/community#1642] [Help] [answered]
### Summary

I'm getting this error: "ELIFECYCLE  Command failed with exit code 1." while I'm trying to deploy my Next.js project. I'm using pnpm, Next.js and Vercel. I've never run into this error before, and couldn't find any helpful resources online about it. Let me know if I should provide any more context or info! :)

### Additional information

Here is the full error message:

```
/vercel/path0/node_modules/.bin/../node/bin/node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /vercel/path0/node_modules/.bin/../node/bin/node)
--
19:49:51.008 | /vercel/path0/node_modules/.bin/../node/bin/node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /vercel/path0/node_modules/.bin/../node/bin/node)
19:49:51.011 | ELIFECYCLE  Command failed with exit code 1.
19:49:51.031 | Error: Command "pnpm run build" exited with 1
```

Here's the root of my package.json file:

```json
{
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@sentry/cli": "^2.13.0",
    "@sentry/nextjs": "^7.38.0",
    "@vercel/analytics": "^0.1.10",
    "@vercel/og": "^0.1.0",
    "autoprefixer": "^10.4.13",
    "csstype": "^3.1.1",
    "million": "1.12.6-beta.2",
    "next": "^13.2.1",
    "node": "^19.6.1",
    "postcss": "^8.4.21",
    "react": "^18.2.0",
    "react-modal": "^3.16.1",
    "sharp": "^0.31.3",
    "tailwindcss": "^3.2.7"
  },
  "devDependencies": {
    "@types/node": "^18.14.2",
    "@types/react": "^18.0.28",
    "eslint": "^8.35.0",
    "eslint-config-next": "^13.2.1",
    "eslint-plugin-react-hooks": "^4.6.0",
    "typescript": "^4.9.5"
  },
}
```

I don't get this error when building locally, and I'm only running into it after trying to deploy my project.

**Comment by amyegan** [comment]
This could be caused by a sub-dependency. Does it work if you [change your Node.js version](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version) to 16.x?

**Reply by tobyab** [reply]
No - I'm still getting the same error

**Comment by tobyab** [comment, marked as answer]
Issue solved! I had to migrate from PNPM to Yarn, and it started deploying with no problem!
